site/app/templates/polls/PollForm.twig

```diff
@@ -202,6 +221,36 @@
         }
         submitErrorChecks();
     };
+    window.onload = function() {
+        toggleTimerInputs();
+    };
+    function toggleTimerInputs() {
+        var enableTimerCheckbox = document.getElementById("enable-timer");
```

Comment:
It would be better to use jQuery instead of manually querying the DOM.  In this case, it would look something like this: `$('#enable-timer')` instead of `document.getElementById("enable-timer")`.

Apply the same throughout the following function.

Reply:
changed the function to use jquery